components/ContinentScreen/ContinentScreen.utils.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Given an array of countries, return the number of unique languages
+ *
+ * @param {object[]} [countries]
+ * @returns {number}
+ */
+export function numberOfLanguages(countries) {
+
+  return new Set( // remove duplicates
+    countries?.map( // in each country
```

Comment:
so this is a pretty heavy piece of code, a map within a map followed by filters etc. You could likely rewrite it to use a single .reduce func. Generally any time you need to map over data and filter it a reduce will do it best

Reply:
I couldn't find a way to use `reduce()`. I may be misunderstanding something, but it doesn't seem to improve the code here. 
Instead, I'll opt for using two forEach loops, which ends up making the code look better.